### PR TITLE
Add doc for monitor datasource

### PR DESCRIPTION
**Pull request by zippolyte**

### CODE REVIEW

File: website/docs/d/monitor.html.markdown [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+layout: "datadog"
+page_title: "Datadog: datadog_monitor"
+sidebar_current: "docs-datadog-datasource-monitor"
+description: |-
+  Retrieve information about an existing monitor.
+---
+
+# datadog_ip_ranges
```
Comment (nmuesch) [inline website/docs/d/monitor.html.markdown:9]:
```suggestion
# datadog_monitor_source
```

Comment (zippolyte) [reply website/docs/d/monitor.html.markdown:9]: Oops 🙊

**Comment by therve** [comment]
I think you need to reference it in `datadog.erb` too.

File: website/datadog.erb [hunk 2]
```diff
@@ -16,6 +16,9 @@
             <li<%= sidebar_current("docs-datadog-datasource-ip-ranges") %>>
               <a href="/docs/providers/datadog/d/ip_ranges.html">datadog_ip_ranges</a>
             </li>
+            <li<%= sidebar_current("docs-datadog-datasource-monitor") %>>
+              <a href="/docs/providers/datadog/d/monitor.html">datadog_ip_ranges</a>
```
Comment (therve) [inline website/datadog.erb:20]:
```suggestion
              <a href="/docs/providers/datadog/d/monitor.html">datadog_monitor</a>
```

**Pull request merged**